src/events/select.ts

```diff
@@ -8,19 +8,15 @@ import {
 
 export function selectHandler(scrollbar: I.Scrollbar) {
   const addEvent = eventScope(scrollbar);
-  const {
-    containerEl,
-    contentEl,
-    offset,
-    limit,
-  } = scrollbar;
+  const { containerEl, contentEl, offset } = scrollbar;
 
   let isSelected = false;
   let animationID: number;
 
   function scroll({ x, y }) {
     if (!x && !y) return;
 
+    let limit = scrollbar.limit;
```

Comment:
Ha! You found a potential bug here!

Although the `scrollbar.offset` object itself will never change (only the value `offset.x` and `offset.y` change), I think it's a good practice to use the newest `offset` and `limit` here, i.e.
```js
const { offset, limit } = scrollbar;
```